### PR TITLE
[PP-5] Create app theme variables

### PR DESCRIPTION
**Pull request by MakFill**
**Task**: https://andrei-lysiuk.atlassian.net/browse/PP-5
**Description**: Create app theme variables

**Added**:
- added scss variables file with all variables that could be used in FE part of app
- added font-family
- added @import-normalize to style.scss

### CODE REVIEW

**Review by Praddby — APPROVED** [review]

**Review by andreikaptsiuh — APPROVED** [review]

**Review by Praddby — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (mrakofff) [inline .vscode/settings.json]: add new pattern to `.gitignore`: `.vscode` and remove this files from PR

File: .vscode/custom.css-data.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (mrakofff) [inline .vscode/custom.css-data.json]: Should be in `.gitignore`

File: src/style.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap');
+
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 14px;
```
Comment (mrakofff) [inline src/style.scss]: Add Global size as `10px` and body `font-size: 1.4rem`

File: normalize.css [hunk 4]
```diff
@@ -0,0 +1 @@
+@import-normalize;
```
Comment (mrakofff) [inline normalize.css]: it should be imported in global `style.css/scss` file not through the VScode

**Review by mrakofff — CHANGES_REQUESTED** [review]
Add some comment to fix

File: .vscode/settings.json (hunk 1 above)
Comment (MakFill) [reply .vscode/settings.json]: added

File: .vscode/custom.css-data.json (hunk 2 above)
Comment (MakFill) [reply .vscode/custom.css-data.json]: deleted

File: src/style.scss (hunk 3 above)
Comment (MakFill) [reply src/style.scss]: html {
font-size: 10px;
}

body {
font-size: 1.4rem
}

File: normalize.css (hunk 4 above)
Comment (MakFill) [reply normalize.css]: done

**Review by mrakofff — APPROVED** [review]

**Pull request merged**